client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -144,7 +144,7 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 
 							<div>
 								<TextControl
-									label={ __( 'Postcode' ) }
+									label={ __( 'Postcode (optional)' ) }
```

Comment:
This change was not part of the PR that was approved last week and looks like was added just before merge and unreviewed? In the future, let's please separate out changes that are unrelated to the purpose of a PR and avoid adding them unreviewed. I welcome a discussion around whether we should add the label or not, but let's do give ourselves that chance at least.

I didn't add an (optional) suffix to the label in #60321 because it's not actually optional for a lot of countries. We just don't want it to be blocking.